internal/provider/modifiers.go

```diff
@@ -0,0 +1,46 @@
+package provider
```

Comment:
Lawl... the removed Default so that you have to write your own? 🤦🏼

Reply:
hahahaha.. exactly what I thought, but I also understand that it gives you more flexibility to implement your own logic.

Reply:
At least have a basic subset of often used ones like default -_-